### PR TITLE
Completed Images (Part 1) 

### PR DESCRIPTION
**Pull request by Rafael805**
Introduced a simple approach to supporting images by automatically replacing URLs that end with an image extension (like .jpg or .png) with an  element.

Modified the text entered by the user before displaying it using regular expressions and then styled the image to properly fit the screen 👍

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -17,6 +17,28 @@
...
+  text-align: center;
+}
+
+p {
+  font-family: 'Muli', sans-serif;
+  margin-bottom: 15px;
+  font-size: 20px;
+}
+
+#teamName {
```
Comment (jchau9) [inline src/main/webapp/css/main.css]: Add tab space

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -17,6 +17,28 @@
...
+color: #CEF0D4;
+font-family: 'Rouge Script', cursive;
+font-size: 80px;
```
Comment (jchau9) [inline src/main/webapp/css/main.css]: add indentation

File: src/main/webapp/css/main.css [hunk 3]
```diff
@@ -17,6 +17,28 @@
...
+font-weight: normal;
+line-height: 100px;
+padding-top: 10px;
+margin: 0 0 50px;
+text-align: center;
+text-shadow: 1px 1px 2px #082b34;
```
Comment (jchau9) [inline src/main/webapp/css/main.css]: add indentation to the css under #teamName

Comment (Rafael805) [reply src/main/webapp/css/main.css]: Done 👍

**Pull request merged**